lib/commands/web.js

```diff
@@ -773,9 +773,13 @@ const extensions = {
   async waitForAtom(promise) {
     const timer = new timing.Timer().start();
 
+    let atomWaitTimeoutMs = ATOM_WAIT_TIMEOUT_MS;
```

Comment:
this could be a ternary operator

Reply:
done [f8b4056](https://github.com/appium/appium-xcuitest-driver/pull/2335/commits/f8b40565030a43bb6fbec4ced6b4439be81d4f39)